conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class FrugallyDeepConan(ConanFile):
+    name = "frugally-deep"
+    version = "0.1-p0"
+    license = "The MIT License (MIT)"
+    url = "https://github.com/Dobiasd/frugally-deep"
+    description = "Use Keras models in C++ with ease"
+    no_copy_source = True
+    exports_sources = ["LICENSE"]
+    requires = ("eigen/3.3.4@conan/stable",
+                "functionalplus/0.2@conan/stable",
```

Comment:
Change here the reference of FunctionalPlus to match`package_name/package_version@user/channel` scope: 
`functionalplus/v0.2-p1@dobiasd/stable` (You can check package reference here https://bintray.com/dobiasd/public-conan/functionalplus%3Adobiasd/v0.2-p1%3Astable)